app/redux/actions/setTokenIdAction.ts

```diff
@@ -0,0 +1,17 @@
+import { TokenIds } from '../../constants/app';
+
+export const SET_TOKEN_ID = 'SET_TOKEN_ID';
+
+export type SetTokenIdAction = {
+  type: 'SET_TOKEN_ID';
```

Comment:
can this be `type: SET_TOKEN_ID` as opposed to `type: 'SET_TOKEN_ID'`?

Reply:
it can't, weirdly enough. I don't remember the exact error, but TS was complaining about thing.